Add onsite volunteers page and links (#1247)
EuroPython/website: src/content/pages/volunteers.mdx

---
title: Volunteers
subtitle: We need your help to make EuroPython 2025 a success!
---

> # Volunteering@EuroPython

Reviewer: Please use one H1 "#" level, and then H2 "##", H3 "###" to keep proper hierarchy.

Author: Fixed now - hopefully all of them

> Python owes much of its success to open source and the community surrounding it. In the same spirit, EuroPython is organized and run by volunteers from the Python community. Alas, we're only a few and need your help to make the show run as smoothly as possible. Each year we are joined by our awesome attendees as volunteers to help in running this wonderful conference. This page explains some of the ways you can help us.

Reviewer 2: UK spelling?
```suggestion
Python owes much of its success to open source and the community surrounding it. In the same spirit, EuroPython is organised and run by volunteers from the Python community. Alas, we're only a few and need your help to make the show run as smoothly as possible. Each year we are joined by our awesome attendees as volunteers to help in running this wonderful conference. This page explains some of the ways you can help us.
```

Being a volunteer means giving your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference. It's a great way to meet new people, learn new things, and have a lot of fun. You can volunteer for a few hours or for the whole conference, it's up to you.

We will also have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.

# Things to remember

> * This is in person volunteering and would require you to be present at the venue.

Reviewer 2: ```suggestion
* This is in-person volunteering and requires you to be present at the venue.
```

* While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
* To enter the conference you need a ticket.
* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.

# Perks for volunteers

> * Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)

Reviewer 2: ```suggestion
* Get an awesome EuroPython volunteer T-shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-)
```

* Be part of a great team and make new friends.

> * A special group photo with all volunteers (time will be announced on the #onsite-volunteers on discord)

Reviewer 2: ```suggestion
* A special group photo with all volunteers (time will be announced on the #onsite-volunteers channel on Discord)
```

# What Volunteering at EuroPython looks like

As a part of EuroPython volunteers' fleet, you will be the face of the conference. Your prime directive will be to ensure everyone coming to the conference has a great time.
We want volunteers to be welcoming, helpful, collaborative, and overall making sure everyone (including yourself) is comfortable.
Despite the different tasks laid down below, volunteers represent a single unit. Sometimes this would mean that as a session chair you will be asked about where the quiet room is. We expect you to help this person out or point them to someone who can.

We've put together an outline of some of the ways you can volunteer your time at the conference.

## Registration Desk

This is a very people-first position and will require spending a considerable time at the registration desk talking to attendees, along with carrying out the following activities:

> * Handing out registration badges
> * Checking student IDs
> * Answering general questions.

Reviewer 2: ```suggestion
* Handing out registration badges
* Checking student IDs
* Answering general questions
```

We want to call out that volunteers at the registration desk will generally not be able to attend talks for the duration of the task.

## T-Shirt Handout

> * Handing out the conference T-shirt
> * This job involves standing and lifting light items.

Reviewer 2: ```suggestion
* Handing out the conference T-shirt
* This job involves standing and lifting light items
```

We want to call out that this job is located at the registration desk, and you will generally not be able to attend talks for the duration of the task.

## Room Manager

A room manager is responsible for ensuring everything in a room runs smoothly for the entirety of a slot. Some of the activities a room manager is expected to carry out involve:

> * Insuring people are seated before a talk, and the room is not overcrowded

Reviewer 2: ```suggestion
* Ensuring people are seated before a talk, and the room is not overcrowded
```

* Making sure the session chair and speakers are in the room in time
* Keeping in touch with the AV team to ensure everything is in order
* Keeping the Code of Conduct (CoC) in mind and contact CoC committee members if needed

We want to call out that room managers might not be able to attend or focus on talks for the duration of the task.

## Session Chairing

A session chair plays a key role in introducing talks and ensuring a seamless transition between talks scheduled during a slot. The activities that a session chair is expected to carry out include:

* Helping speakers in preparing their computers for the talks
* Introducing the speakers to the audience
* Signalling time left to the speakers
* Making sure talks start and finish on time

> * Moderating Q&A sessions, assisting with passing the microphones and read out questions from remote attendees

Reviewer 2: ```suggestion
* Moderating Q&A sessions, assisting with passing the microphones and reading out questions from remote attendees
```

## Setup/Teardown

> * Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)

Reviewer 2: ```suggestion
* Putting up signage in hallways for various provisions (toilets, quiet room, childcare etc.)
```

* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward

> * This job involves walking and carrying items, some of which may be heavy.

Reviewer 2: ```suggestion
* This job involves walking and carrying items, some of which may be heavy
```

## Greeter / Badge check

> * Checking attendees on the venue for badges.
> * Send people to the registration desk if they don't have one and are not from the venue (e.g. AV team, catering, etc.).
> * Report people that don't have badges and don't want to go to the registration desk.

Reviewer 2: ```suggestion
* Checking attendees on the venue for badges
* Sending people to the registration desk if they don't have one and are not from the venue (e.g. AV team, catering, etc.)
* Reporting people that don't have badges and don't want to go to the registration desk
```

## Runners and Ad-hoc Activities

There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):

* Checking rooms to ensure all the necessary resources are in place

> * Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties

Reviewer 2: ```suggestion
* Helping out with registration desk duties, T-shirt handout, or room manager duties
```

* Moving chairs, tables, or poster walls

> * Arranging transfers between venue and airport, etc. (Rarely)

Reviewer 2: ```suggestion
* Arranging transfers between venue and airport, etc. (rarely)
```

# How to sign up

We are currently still working out the latest details for the volunteering program. (We are trying to get more perks enabled for your volunteering time).

> Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu) .

Reviewer 2: ```suggestion
Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu).
```

<div class="text-center mb-8">

<ButtonLink href="mailto:volunteers@europython.eu">volunteers @ europython.eu</ButtonLink>
</div>

# More information

> ## Why be a EuroPython Volunteer?

Reviewer 2: Perhaps?
```suggestion
## Why volunteer at EuroPython?
```

* To help the EuroPython organizers keep the conference affordable for everyone
* Show your EuroPython support to the Python community (by wearing your exclusive T-Shirt)
* Get a huge applause during the closing session of the conference

> * Get your name on our EuroPython 2025 Team page to stay there forever.

Reviewer 2: ```suggestion
* Show your EuroPython support to the Python community (by wearing your exclusive T-shirt)
* Get a huge applause during the closing session of the conference
* Get your name on our EuroPython 2025 Team page to stay there forever
```

## How much work is involved?

* There are different tasks with different time required

> * Each task is related with the concept of slot (Usually bracketed by the breaks)

Reviewer 2: ```suggestion
* Each task is related with the concept of slot (usually bracketed by the breaks)
```

> * You can sign up to any number of them.
> * We would really appreciate it if you can sign up for more than one slot.

Reviewer 2: ```suggestion
* You can sign up to any number of them
* We would really appreciate it if you can sign up for more than one slot
```

## What kind of qualifications do I need?

* You should be able to speak some English.
* If you speak additional languages, even better. Please let us know.

> * Have time between two talks? Then why not spend it as a EuroPython Volunteer on the spot?

Reviewer 2: ```suggestion
* Have time between two talks? Then why not spend it as a EuroPython volunteer on the spot?
```

## Ask for help

> If you run into any problems, please ask in the #onsite-volunteers channel in discord or contact our volunteers@europython.eu for any information you may need.

Reviewer 2: ```suggestion
If you run into any problems, please ask in the #onsite-volunteers channel in Discord or contact our volunteers@europython.eu for any information you may need.
```
